strongbox-web-core/src/test/resources/ldap/strongbox.ldif

```diff
@@ -1,109 +0,0 @@
-# Depending on the server, this may already exist. 
```

Comment:
How are you pre-loading test users in LDAP...?

Reply:
it's in the `strongbox-ldap-authentication-provider/src/test/resources/ldapServerApplicationContext.xml`.
`security:ldap-server` allows to setup embedded LDAP with `strongbox.ldif` contents.